app/api/v2/models/incident.py

```diff
@@ -140,6 +143,48 @@ def get_incidents(self):
             return self.message["NOT_FOUND"] + str(id) \
              + "Record could not be found or doesnot exist"
 
+    def update_incident(self, id, prop, prop_value):
+        """
+        Update a property of an incident
+        by taking the :id arg to find the incident record,
+        :prop arg to identify the key of the property of the record to update,
+        :prop_value arg to store the new property value
+
+        Returns
+        --------
+        dictionary
+            dictionary containing all details of found incident and a success message
+
+            OR
+
+        dictionary
+            dictionary contaiing an error message and the success status of the update.
+        """
+        incident = self.get_single_incident_by_id(id)
+        if not isinstance(incident, dict):
+            return self.message["NOT_FOUND"] + str(id) \
+             + " Record could not be found or doesnot exist"
+
+        # if incident['status'] != "draft":  # or auth'd user is not admin
```

Comment:
One of our fellow boot campers was advised by their LFA that comments should be strictly descriptive comments. You should never comment out code but rather, consider removing it all at once if it is not required.

Reply:
Definitely! Going to remove this one thanks!